src/PureCloud.Client/Apis/LearningApi.cs

```diff
@@ -51,56 +296,492 @@ public async Task<LearningModule> GetLearningModuleAsync(string moduleId, IEnume
 
         if (expand?.Any() == true)
         {
-            parameters.Add("expand", string.Join(",", expand));
+            parameters.Add("expand", string.Join(",", expand.Select(UriHelper.ParameterToString)));
```

Comment:
Same here add multiple expand values to the collection and don't use .Any() just null exception